Makefile

```diff
@@ -15,6 +16,8 @@ ALL_PKGS := $(shell go list $(sort $(dir $(ALL_SRC))))
 ALL_DOC := $(shell find . \( -name "*.md" -o -name "*.yaml" \) \
                                 -type f | sort)
 
+TOOLS_MOD_DIR := ./internal/tools
```

Comment:
Use TOOLS_MOD_DIR in line 4?

Reply:
A bit ugly to have `"$(TOOLS_MOD_DIR)/*"`. Your call, we can do a separate PR.